notebooks/1.0-vse-eda-rooms.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>You don't need to always print what you want to show in notebooks. If you use just:</p><p><code>df.groupby('BsmtFullBath')['SalePrice'].describe()</code></p><p>the statics will be shown to the user in a better format. You should use the print function just when you want to display custom messages to the user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Wow, didn't know about it. It's way better like this!</p>